### PR TITLE
Prepare project for Heroku deployment

### PR DESCRIPTION
**Pull request by waxlamp**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,8 +19,21 @@
...
+        "requests",
+        "webargs",
+        "flask",
+        "flask-cors",
+        "flasgger",
+        "typing-extensions",
+        "uwsgi",
+        "sentry-sdk[flask]",
+        "gunicorn",
+    ],
```
Comment (AlmightyYakob) [inline setup.py]: Why are these needed here? Since we already have the `requirements.txt`

Comment (waxlamp) [reply setup.py]: Long story but read this article: https://caremad.io/posts/2013/07/setup-vs-requirement/.

I chatted with David Manthey about it this morning, and the bottom line is, this is the technically correct way to express the "install requirements", and is indeed redundant with the requirements.txt file, but for completeness I did it this way.

File: setup.py [hunk 2]
```diff
@@ -21,8 +19,21 @@
         "Programming Language :: Python :: 3.6",
```
Comment (JackWilb) [inline setup.py]: I know you haven't changed these in this PR, but why are there versions of python here that are incompatible with our app?

Comment (waxlamp) [reply setup.py]: Good catch, I will fix that on this branch.

Comment (waxlamp) [reply setup.py]: See 883b3c5
